Makefile

```diff
@@ -66,4 +79,11 @@ cargo_version_check:
 		     exit 1; \
 		fi
 
+# A target that sets up directories the bwrap sandbox needs.
+.PHONY: sandbox_setup
+sandbox_setup:
+	mkdir -p build
```

Comment:
Do we need to explicitly specify the parent directory here or is it guaranteed that we're in the right directory?

Reply:
`make` always executes actions from the same directory as the Makefile itself. e.g. if you do `make -C a/b/c/d/Makefile` to run a Makefile in a directory different from the present working directory, `make` will switch to `a/b/c/d` before running its actions.

So yes, this is guaranteed by `make`.